centos_ci.sh

```diff
@@ -208,6 +216,52 @@ function docs_tar_upload() {
   echo "Find docs tar here http://artifacts.ci.centos.org/minishift/minishift/docs/$LATEST."
 }
 
+function bump_to_release_version() {
+  # Set Terminal
+  export TERM=xterm-256color
+  # Add git a/c identity
+  git config user.email "admn.minishift@gmail.com"
+  git config user.name "Minishift Bot"
+  # Export GITHUB_ACCESS_TOKEN
+  export GITHUB_ACCESS_TOKEN=$GITHUB_TOKEN
+  # Create master branch as git clone in CI doesn't create it
+  git checkout -b master
+  # Bump version and commit
+  sed -i "s|MINISHIFT_VERSION = .*|MINISHIFT_VERSION = $RELEASE_VERSION|" Makefile
+  git add Makefile
+  git commit -m "cut v$RELEASE_VERSION"
+  # Push update
+  # TODO: Change to minishift once final review is done
+  git push https://minishift-bot:$GITHUB_TOKEN@github.com/minishift-bot/minishift master
+}
+
+function add_release_notes() {
+  # TODO: Change to minishift once final review is done
+  id=$(curl -s "https://api.github.com/repos/minishift-bot/minishift/releases" | jq --arg release $RELEASE_VERSION -r ' .[] | select(.name | test([$release])) | .id')
```

Comment:
Maybe make 'minishift-bot/minishift' a variable, seems it is needed at several places.